packages/icons-ui/README.md

```diff
@@ -1,6 +1,10 @@
 ## Description
 
-[Spectrum UI Icons](https://spectrum.adobe.com/page/icons/) delivered in a flexible template tag so that they can be leveraged across various frameworks. The default export of this package pre-applies the `html` template tag from `lit-html` for ease of use in the Spectrum Web Components library. Please remember to consult Spectrum's [Iconography Guidelines](https://spectrum.adobe.com/page/iconography/) when planning how to leverage these icons in the visual delivery of your application. For technical information on using these iconos in projects powered by various javascript frameworks, check out the "Extended use cases" sectino below.
+[Spectrum UI Icons](https://spectrum.adobe.com/page/icons/) delivered as registered custom elements (e.g. `<sp-icon-arrow75>`), unregistered class extensions of `LitElement` (e.g. `IconArrow75`), or a flexible icons function (e.g. `Arrow75Icon`) with a customizable template tag so that they can be leveraged across various frameworks.
```

Comment:
The class is actually how you might leverage these classes with a scoped registry and while I don't have any reason to extend a class, it could happen. Say you wanted to mixin some styles or similar.

This content structure does make sense!